### PR TITLE
[Issue 66] - Migrating from TSLint to ESLint

### LINKED ISSUE #66: Migrate from TSLint to ESLint
Because [TSLint](https://www.npmjs.com/package/tslint) is now deprecated in favor of [ESLint](https://eslint.org/), we should migrate to use ESLint.

We may be able to do this quickly by using https://github.com/typescript-eslint/tslint-to-eslint-config.

### PR DESCRIPTION
**Pull request by natesulat**
Signed-off-by: Nate Sulat <nates@kiva.org>

### CODE REVIEW

**Comment by natesulat** [comment]
Issue 66?
image

File: tsconfig.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "esModuleInterop": true
   },
   "include": [
-    "src/**/*"
```
Comment (ghost) [inline tsconfig.json:21]: Presumably we want to include `src/**/*` too? Also, I don't think we want to include tests in the compiled source.

**Review by matt-raffel-kiva — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (natesulat) [reply tsconfig.json:21]: There's no `src` directory in this repo. And we don't have a `main` key defined in `package.json`, so there's no compilation target that someone could consume.

Happy to create a `tsconfig.eslint.json` file for this directory, but it seemed like overkill.

Comment (ghost) [reply tsconfig.json:21]: Ahh sorry, I had an old version of this pulled down locally, which had `src/` still.

**Review by ghost — APPROVED** [review]

**Pull request merged**